src/Client/BigQuery/Retry.php

```diff
@@ -28,7 +28,7 @@ public static function getRetryDecider(LoggerInterface $logger): callable
         return static function (Throwable $ex) use ($logger): bool {
             $statusCode = $ex->getCode();
 
-            if (in_array($statusCode, [429, 500, 503,])) {
```

Comment:
To je problém proč padaly testy :) 401 sa nedělal retry, problém je, že teď sa dělá takže když dojdů vadné credentials tak sa to bude pár minut točit, bude potřeba to ošetřit v connection a v sapi testech jinak to bude zdržovat.